src/controllers/authenticationController.ts

```diff
@@ -94,8 +98,8 @@ export default class AuthenticationController {
     const user = errorOrUser.value;
 
     const errorOrResponse = await this.sessionStorage.del(
-      user.session_token,
-      user.wallet_token
+      user.session_token as SessionToken,
```

Comment:
`user.session_token` should already be of type `SessionToken`, it should be `extractUserFromRequest` that does the valitation